### PR TITLE
docs(tekton): updated images

### PR DESCRIPTION
**Pull request by hmanwani-rh**
GitHub issue: https://github.com/janus-idp/backstage-plugins/issues/394

### CODE REVIEW

**Comment by debsmita1** [comment]
@hmanwani-rh The Janus showcase app has been updated. Could you re-capture the PipelineRun list page ?

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

**Comment by hmanwani-rh** [comment]
> @hmanwani-rh The Janus showcase app has been updated. Could you re-capture the PipelineRun list page ?

@debsmita1 I've updated the screenshot, please take a look one more time.

**Comment by debsmita1** [comment]
> > @hmanwani-rh The Janus showcase app has been updated. Could you re-capture the PipelineRun list page ?
> 
> @debsmita1 I've updated the screenshot, please take a look one more time.

Could you also add a screenshot of the page when you click on a pipeline run in the list page (it will open the pipelinerun's visualization without the cluster dropdown) ?

File: plugins/tekton/README.md [hunk 1]
```diff
@@ -172,4 +172,4 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
 
-   ![tekton-tab](./images/tekton-plugin-user2.png)
+    ![tekton-tab](./images/tekton-plugin-user2.png)
```
Comment (invincibleJai) [inline plugins/tekton/README.md:175]: job is failing https://github.com/janus-idp/backstage-plugins/actions/runs/5877416693/job/15937509561?pr=606 due to formatting issue from prettier for this

Comment (hmanwani-rh) [reply plugins/tekton/README.md]: Done

Comment (hmanwani-rh) [reply plugins/tekton/README.md]: The job is still failing

File: plugins/tekton/README.md [hunk 2]
```diff
@@ -162,14 +162,15 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
...
    ![ci-cd-tab-tekton](./images/tekton-plugin-user1.png)
 
    There is also a **GO TO TEKTON** option at the bottom, which redirects you to the **TEKTON** tab.
 
-1. Click **GO TO TEKTON** or select the **TEKTON** tab in the entity view page.
+1. Click **GO TO TEKTON** or select the **TEKTON** tab on the entity view page.
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
+   
+   ![usertekton-tab](./images/tekton-plugin-user2.png)
```
Comment (tumido) [inline plugins/tekton/README.md:176]: Please remove the empty line at the end of the file to make it pass the CI,

or run `yarn prettier:fix` locally to fix it automatically.

**Comment by debsmita1** [comment]
@hmanwani-rh Could you run `yarn prettier:fix` and push again ?

**Comment by hmanwani-rh** [comment]
> @hmanwani-rh Could you run `yarn prettier:fix` and push again ?

Hi Debsmita,
I did try this and it is showing me the following error:

heenamanwani@hmanwani-mac backstage-plugins % yarn prettier:fix      
yarn run v1.22.19
$ prettier --write .
[error] Invalid configuration for file "/Users/heenamanwani/Documents/repos/backstage-plugins/.eslintrc.js":
[error] Cannot find module '@spotify/prettier-config'
[error] Require stack:
[error] - /Users/heenamanwani/Documents/repos/backstage-plugins/.prettierrc.js
[error] - /usr/local/lib/node_modules/prettier/internal/internal.mjs
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Could you please help me with this error?

File: plugins/tekton/README.md (hunk 2 above)
Comment (hmanwani-rh) [reply plugins/tekton/README.md]: I did run the command, but it's showing the following error. Not sure, if I'm doing something wrong:

heenamanwani@hmanwani-mac backstage-plugins % yarn prettier:fix
yarn run v1.22.19
$ prettier --write .
[error] Invalid configuration for file "/Users/heenamanwani/Documents/repos/backstage-plugins/.eslintrc.js":
[error] Cannot find module '@spotify/prettier-config'
[error] Require stack:
[error] - /Users/heenamanwani/Documents/repos/backstage-plugins/.prettierrc.js
[error] - /usr/local/lib/node_modules/prettier/internal/internal.mjs
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

File: plugins/tekton/README.md [hunk 3]
```diff
@@ -172,14 +172,20 @@ Tekton is a front-end plugin that enables you to view the `PipelineRun` resource
...
    There is also a **GO TO TEKTON** option at the bottom, which redirects you to the **TEKTON** tab.
 
-1. Click **GO TO TEKTON** or select the **TEKTON** tab in the entity view page.
+1. Click **GO TO TEKTON** or select the **TEKTON** tab on the entity view page.
 
    The **TEKTON** tab contains the list of pipeline runs related to a cluster. The list contains pipeline run details, such as **NAME**, **STATUS**, **TASK STATUS**, **STARTED**, and **DURATION**.
 
    ![tekton-tab](./images/tekton-plugin-user2.png)
+
+1. Select a PipelineRun to view the related resources with a Kubernetes cluster.
```
Comment (debsmita1) [inline plugins/tekton/README.md:187]:
```suggestion
2. Click on a PipelineRun name to view the PipelineRun visualization.
```

Comment (hmanwani-rh) [reply plugins/tekton/README.md]: Done, changed the sentence as follows:

Click a PipelineRun name in the list to view the PipelineRun visualization.

**Comment by debsmita1** [comment]
LGTM !
Thanks @hmanwani-rh !

**Comment by hmanwani-rh** [comment]
Merging this PR

**Pull request merged**